tests/bug-1120.sil

```diff
@@ -0,0 +1,15 @@
+\begin[papersize=a6]{document}
```

Comment:
Let's not use SILE document rendering for tests that aren't actually testing the output of typesetting functions; they are a huge maintenance burden to keep fresh compared to more targeted tests. In this case I think a Busted basted spec will suffice since all we want to know is if a string got decoded correctly.

I'll rig it up.